Readme.md

```diff
@@ -31,16 +41,17 @@ For example, if the project you're working is called `Awesome Project`, then you
 
 ## Theme development
 
-- Create a custom solution file on your prefixed development branch as a copy of `DotNest.Core.SDK.sln`, e.g. `AwesomeProject.sln`.
-- From here on, general Orchard Core theme development rules apply with some DotNest-related extras. You can read about all these on the [`Theming a DotNest site` page of the DotNest Knowledge Base](https://dotnest.com/knowledge-base/topics/theming/).
-- If your theme contains Liquid templates, enable the `Liquid` feature for these to be picked up by Orchard Core.
-- The Media Theme on DotNest also has an automated mechanism to include some site-level resources. This might come in handy e.g. if your theme doesn't have a base theme and/or you're not overriding the `Document` or `Layout` shapes.
```

Comment:
I don't see any indication of this actually being implemented, but let me know if it actually is.

Reply:
Me neither. @barthamark did you implement such feature?

Reply:
Or @porgabi , you added this line. Is this just a copy paste error from [https://github.com/Lombiq/DotNest-SDK](https://github.com/Lombiq/DotNest-SDK) ?

Reply:
OK, this is clearly a copy paste error.